internal/parser/parser.go

```diff
@@ -64,15 +64,23 @@ func (p *Parser) Parse(r io.Reader) (*Example, error) {
 	source := string(bytes)
 
 	parseScript := func(s string) []string {
+		const (
+			scriptBegin = "```bash"
+			scriptEnd   = "```"
+		)
+
 		var r []string
-		const begin = "```bash"
-		for start := strings.Index(s, begin); start > 0; start = strings.Index(s, begin) {
-			end := strings.Index(s[start+len(begin):], "```") + start + len(begin)
+		for start := strings.Index(s, scriptBegin); start >= 0; start = strings.Index(s, scriptBegin) {
+			start += len(scriptBegin)
+
+			end := strings.Index(s[start:], scriptEnd)
 			if end < 0 {
 				break
 			}
-			r = append(r, strings.TrimSpace(s[start+len(begin):end]))
-			s = s[end+len("```"):]
+			end += start
+
+			r = append(r, strings.TrimSpace(s[start:end]))
+			s = s[end+len(scriptEnd):]
```

Comment:
Why do we need to change parseScript here?

Reply:
Because line 74 is incorrect:
```go
end := strings.Index(s[start+len(begin):], "```") + start + len(begin)
if end < 0 {
	break
}
```
`end` is never `< 0`.